controllers/territory_controller/tests.py

```diff
@@ -18,29 +20,81 @@
 # Root directory of the SR webots simulator (equivalent to the root of the git repo)
 REPO_ROOT = Path(__file__).resolve().parent.parent.parent
 
+sys.path.insert(1, str(REPO_ROOT / 'modules'))
 
+from sr.robot.radio import StationCode as RadioStationCode  # isort:skip
+
+
+class MockStationCode(str, enum.Enum):
+    T0 = 'T0'
+    T1 = 'T1'
+    T2 = 'T2'
+    T3 = 'T3'
+    T4 = 'T4'
+    T5 = 'T5'
+    T6 = 'T6'
+    T7 = 'T7'
+    T8 = 'T8'
+
+
+@patch('territory_controller.StationCode', MockStationCode)
 class TestAttachedTerritories(unittest.TestCase):
-    'Test build_attached_capture_trees/get_attached_territories'
-
-    _zone_0_territories = {
-        StationCode.BG,
-        StationCode.TS,
-        StationCode.OX,
-        StationCode.VB,
-        StationCode.PL,
+    """
+    Test build_attached_capture_trees/get_attached_territories
+    Uses a fixed, reduced map:
+        z0 ── T0 ── T1 ── T2 ── T3 ── z1
+         │         ╱  ╲                │
+         └─ T4 ── T5 ─ T6 ── T7 ── T8 ─┘
+    """
+
+    _zone_0_territories = {MockStationCode.T0, MockStationCode.T1, MockStationCode.T6}
+    _zone_1_territories = {
+        MockStationCode.T3,
+        MockStationCode.T2,
+        MockStationCode.T5,
+        MockStationCode.T4,
+    }
+    _zone_1_disconnected = {MockStationCode.T4, MockStationCode.T5}
+    _zone_0_capturable = {
+        MockStationCode.T2,
+        MockStationCode.T4,
+        MockStationCode.T5,
+        MockStationCode.T7,
+    }
+    _zone_1_capturable = {MockStationCode.T1, MockStationCode.T8}
+    _zone_0_uncapturable = {MockStationCode.T8, MockStationCode.T3}
+    _zone_1_uncapturable = {
+        MockStationCode.T0,
+        MockStationCode.T4,
+        MockStationCode.T5,
+        MockStationCode.T6,
+        MockStationCode.T7,
     }
-    _zone_1_territories = {StationCode.PN, StationCode.EY, StationCode.PO, StationCode.YL}
-    _zone_1_disconnected = {StationCode.PN, StationCode.EY}
 
     def load_territory_owners(self, claim_log: ClaimLog) -> None:
-        # territories BG, TS, OX, VB, etc. owned by zone 0
         for territory in self._zone_0_territories:
-            claim_log._station_statuses[territory] = Claimant.ZONE_0
+            claim_log._station_statuses[territory] = Claimant.ZONE_0  # type: ignore
```

Comment:
Please could we have an [error code](https://mypy.readthedocs.io/en/latest/error_code_list.html) on this ignore?
I'm guessing that this is due to the use of a different set of station codes?
Should we perhaps make the `ClaimLog` generic so that it can handle different maps?

Reply:
af3ffdb3961af54e82c4f83ed78e3f814b653f55